currencies/rate_converter.go

```diff
@@ -111,6 +116,9 @@ func (rc *RateConverter) Update() error {
 		rc.rates.Store(rates)
 		rc.lastUpdated.Store(time.Now())
 	} else {
+		if rc.CheckStaleRates() {
+			rc.rates.Store((*Rates)(nil))
+		}
 		glog.Errorf("Error updating conversion rates: %v", err)
```

Comment:
Consider adding to the error log that we now fall back to a constant rate conversion.